src/main/java/com/spring/boot/amazon/helper/SimpleReader.java

```diff
@@ -0,0 +1,22 @@
+package com.spring.boot.amazon.helper;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class SimpleReader implements Reader {
+    public List read(String csvFile) {
+        List<String> records = new ArrayList<>();
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(csvFile))) {
+            while (bufferedReader.ready()) {
+                records.add(bufferedReader.readLine());
+            }
+        } catch (IOException e) {
+            throw new RuntimeException(e);
+        }
+        records.remove(0);
```

Comment:
I would recommend you remove the first line (e.g. CSV header) in the CSV parser class, not in the reader. This reader class is general and could be used for reading not only CSV files.

Reply:
ok, thank you!